cfme/utils/ssh.py

```diff
@@ -240,8 +254,9 @@ def run_command(
         logger.info("Running command %r", command)
         if self.is_pod and not ensure_host:
             # This command will be executed in the context of the host provider
-            command = 'oc rsh {} bash -c {}'.format(self._container, quote(
-                'source /etc/default/evm; ' + command))
+            command = 'source /etc/default/evm; ' + command
```

Comment:
**REQUIRED:** This is a confusing series of assignments and formats
```python
            command = 'source /etc/default/evm; ' + command
            cmd = 'oc exec --namespace={proj} {pod} -- bash -c {cmd}'
            command = cmd.format(proj=self._project, pod=self._container, cmd=quote(command))
```

Should become something like this I think

```python
            command_to_run = 'source /etc/default/evm; ' + command
            oc_cmd_string = 'oc exec --namespace={proj} {pod} -- bash -c {cmd}'.format(
                proj=self._project, pod=self._container, cmd=quote(command_to_run)
            )
```